### DISCUSSION TITLE
How did you add the furigana to the Anki decks?

### DISCUSSION THREAD

**Discussion by Fozz13** [SethClydesdale/genki-study-resources#284] [Questions] [answered]
I am trying to modify some of the cards, and I don't understand how you added the furigana.

**Comment by SethClydesdale** [comment, marked as answer]
Hello, @Fozz13!

You can add furigana by using the ``<ruby>`` and ``<rt>`` tags ([MDN link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ruby)). Basic formatting is like this:
```html
<ruby>漢字<rt>かんじ</rt></ruby>
```
